lxd/devices.go

```diff
@@ -184,16 +184,21 @@ func deviceTaskBalance(d *Daemon) {
 	// Get effective cpus list - those are all guaranteed to be online
 	effectiveCpus, err := cGroupGet("cpuset", "/", "cpuset.effective_cpus")
 	if err != nil {
-		shared.Log.Error("Error reading host's cpuset.effective_cpus")
-		return
+		// Older kernel - use cpuset.cpus
+		shared.Log.Warn("Failed reading host's cpuset.effective_cpus, falling back to cpuset.cpus")
```

Comment:
We do have similar detection code in the daemon init function which then sets global variables.
Not sure whether it's warranted in this case though, there is no loss of functionality by having us do the fallback.